File: fundamentals-of-programming/list-3/exercise-12.py
pares = 0
impares = 0

lastNum = 0
while (lastNum != -1):
  lastNum = int(input("Digite um número: "))
  if (lastNum >= 0):
    if (lastNum % 2 == 0):
      pares += 1
    else:
      impares += 1

print(f"Total de pares: {pares}")
print(f"Total de ímpares: {impares}")

File: fundamentals-of-programming/bimester-2/list-1/exercise-12.c
#include <stdio.h>
#include <stdlib.h>

main() {
  int pares = 0, impares = 0, num;

  do {
    printf("Digite um número (-1 para parar): ");
    scanf("%d", &num);

    if (num != -1) {
      if (num % 2 == 0)
        pares++;
      else
        impares++;
    }
  } while (num != -1);

  printf("Quantidade de ímpares: %d\n", impares);
  printf("Quantidade de pares: %d", pares);
}